init.sh

```diff
@@ -1,8 +1,14 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 cd $(dirname $0)
 CONFIG_DIR=${HOME}/.config/ocm-container
 
+if [[ -z ${aliases_file} ]]; then
+	echo "run command with aliases file"
+	echo 'aliases_file=$(alias) '"$0 $@"
+	exit
+fi
```

Comment:
Where is `aliases_file` being set?  What is this block supposed to do?

Reply:
it's for https://github.com/openshift/ocm-container/pull/86/files/515d8ea62181ef5fa3f1f6e84db39f9b40dabba6#diff-408f2ee929058c9e991c5af8d003eaded2c24e8616b94e48de84031a80397c0cR59-R63

as the aliases configured on the system didn't pass through to the bash script, I did this hack

run the `./init.sh`
it will prompt on running as 
```
aliases_file=$(alias) ./init.sh
```

Reply:
So this will import all of your aliases into the init.sh command, but not the container?

Reply:
yeah, and this is used to verify if `ocm-container-stg` is set, as this was one of the steps the init.sh file had before